AppCenterDemoApp/Assets/Puppet/Editor/BuildPuppet.cs

```diff
@@ -13,7 +13,7 @@ public class BuildPuppet
 {
     private static readonly string BuildFolder = "CAKE_SCRIPT_TEMPPuppetBuilds";
     private static readonly string AssetsFolder = "Assets";
-    private static readonly string AppIdentifier = "com.microsoft.appcenter.unity.demo";
+    private static readonly string AppIdentifier = "com.microsoft.appcenter.unity.puppet";
```

Comment:
I guess it should be demo

Reply:
@MatkovIvan I've made change. Please check it. Thanks